scripts/generate-settings-docs.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/env python3
+#!/usr/bin/env python3
```

Comment:
?

Reply:
That's the proper location of `env`.  Was too lazy to open a separate PR for it. 🤷